public/app/core/utils/navBarItem-translations.ts

```diff
@@ -153,8 +153,6 @@ export function getNavTitle(navId: string | undefined) {
       return t('nav.slo.title', 'SLO');
     case 'plugin-page-k6-app':
       return t('nav.k6.title', 'Performance');
-    case 'monitoring':
-      return t('nav.observability.title', 'Observability');
```

Comment:
This is duplicated by another case higher up, which has the same display value (`Observability`)